src/pages/billing/List.js

```diff
@@ -221,12 +247,19 @@ export const BillingList = _.flow(
     }
   })
 
+  checkOwner() {
```

Comment:
Added this check that a user has the role "owner" on at least one Billing Project, so that we only show the message "Select a billing project" if a user can actually view a Billing Project's detail.

@panentheos @petesantos  would you review?

Reply:
nit: does this need to be stored in state (causing an additional render)? Could it just be a ternary that gets executed at render? Once the billing projects are updated it should work correctly